.github/ubiquibot-config.yml

```
@@ -2,3 +2,8 @@ price-multiplier: 1.5
 default-labels:
   - "Time: <1 Hour"
   - "Priority: 1 (Normal)"
+new-contributor-greeting:
+  enabled: true
+  header:  "Thank you for contributing to UbiquiBot! Please be sure to set your wallet address before completing your first bounty so that the automatic payout upon task completion will work for you."
+  helpMenu: true
```

```suggestion
  help-menu: true
```

yaml should be in kebab-case

---

I just have a gut feeling that when it is read by the getNewContributorConfig it might throw some error

---

Yeah you're right, this will be a problem because internal config uses camelCase and yaml config uses kebab-case, so this will cause trouble in the future too. I think it would be best to stick to one format for both configs - camelCase is natural in JS/TS so I think it should be also used in yaml config.
I will make a new issue for this but for now modify `getNewContributorConfig` to read from kebab-case and convert to camelCase manually.